lib/chef/knife/cloud/google_service.rb

```diff
@@ -186,12 +186,13 @@ def list_project_quotas
     def validate_server_create_options!(options)
       raise "Invalid machine type: #{options[:machine_type]}" unless valid_machine_type?(options[:machine_type])
       raise "Invalid network: #{options[:network]}" unless valid_network?(options[:network])
+      raise "Invalid subnet: #{options[:subnet]}" if options[:subnet] && !valid_subnet?(options[:subnet])
       raise "Invalid Public IP setting: #{options[:public_ip]}" unless valid_public_ip_setting?(options[:public_ip])
       raise "Invalid image: #{options[:image]} - check your image name, or set an image project if needed" if image_search_for(options[:image], options[:image_project]).nil?
     end
 
     def check_api_call(&block)
-      block.call
```

Comment:
I *thought* it was more idiomatic to use `block.call` (and is what I'm accustomed to historically) but a recent change in rubocop/chefstyle has this being a thing to address. I'll just omit the block var to be consistent as, functionality-wise for this gem, it just don't matter. :)